chord_drs/routes.py

```diff
@@ -29,6 +31,8 @@
 
 drs_service = Blueprint("drs_service", __name__)
 
+path_for_git = os.path.abspath(os.path.join(os.path.dirname(__file__), ".."))
```

Comment:
Consider using a variable name such as `app_dir` or `app_path`. `path_for_git` might be confused with the path to the actual git executable.